lib/evaluate.js

```diff
@@ -366,6 +369,27 @@ export const runFraudDetection = async (roundIndex, measurements, sparkRoundDeta
     }
   }
 
+  //
+  // 3. Group measurements to per-task committees and find majority result
```

Comment:
This might change to `per-task per-field committees`, depending on how discussion above goes